torchao/prototype/awq/api.py

```diff
@@ -105,7 +105,10 @@ def _awq_transform(
     dummy_mod = DummyModule(observed_linear.weight * equalization_scale)
     quant_mod = base_config_handler(dummy_mod, config.base_config)
     qw = quant_mod.weight
-    qw = to_weight_tensor_with_linear_activation_scale_metadata(qw, equalization_scale)
```

Comment:
seems like we should deprecate `WeightTensorWithLinearActivationScaleMetadata` in favor of the mixin? Can we file an issue for that?